packages/server/lib/experiments.ts

```diff
@@ -68,9 +69,10 @@ const _summaries: StringValues = {
   ```
 */
 const _names: StringValues = {
-  experimentalFetchPolyfill: 'Fetch polyfill',
+  experimentalFetchPolyfill: 'Fetch Polyfill',
   experimentalInteractiveRunEvents: 'Interactive Mode Run Events',
-  experimentalSourceRewriting: 'Improved source rewriting',
+  experimentalLoginFlows: 'Login Flows',
+  experimentalSourceRewriting: 'Improved Source Rewriting',
   experimentalStudio: 'Studio',
 }
```

Comment:
hmmm not in scope, but we should def pull these from the config package some day.